README.md

```diff
@@ -40,6 +40,7 @@ make
   Consul API queries to gather all information about each service. Health check
   information are available via `consul_health_service_status` as well, but
   only for services which have a health check configured. Defaults to true.
+* __`consul.health-status-code`:__  Returns a health status code for passing (0), warning (1) or critical (2) via `consul_health_service_status` and `consul_catalog_service_node_healthy`. Default health status code is passing (1) and critical/warning (0).
```

Comment:
What I meant was that with the current staleness handling of time series (when a target stops populating a time series, the server keeps reporting the last value for a fix timeout of 5m by default), just changing the label name will cause issues. So just changing `my_metric{status="warning"} 1` at T1 to `my_metric{status="passing"} 1` at T2 will result `my_metric{status="warning"} == 1` being true. So it's necessary to always export all labels states: `my_metric{status="passing"} 0\nmy_metric{status="warning"} 1\nmy_metric{status="critical"} 0\n` at T1 etc.